case-studies/investment-curve/case-study.html

```diff
@@ -24,7 +25,7 @@
 
 ## The Investment Curve
 
-As much as we love the technology, we can not escape the financial aspect of cryptocurrency.  When hype takes hold price movement, both up and down, come swiftly and boldly.  These movements aren't too foreign to the markets and have actually been well studied.  Dr. Jean-Paul Rodrigue of Hofstra University created the a diagram to depict the change in sentiment and interest of an asset which, according to his research, are the fundamental causes of the extreme volatility. As part of our R course, in this brief case study we'll walk you through the recreation of Dr. Rodrigue's famous graph using the ggplot2 library.
+As much as we love the technology, we can not escape the financial aspect of cryptocurrency.  When hype takes hold, price movements, both up and down, come swiftly and boldly.  These movements aren't too foreign to the markets and have actually been well studied.  Dr. Jean-Paul Rodrigue of Hofstra University created the a diagram to depict the change in sentiment and interest of an asset which, according to his research, are the fundamental causes of the extreme volatility. As part of our R course, in this brief case study we'll walk you through the recreation of Dr. Rodrigue's famous graph using the ggplot2 library.

```

Comment:
@omnianalytics do you think movements should be plural here? I can see an argument for singular and plural.